best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
```

Comment:
I'm also  confused by the naming here. I've personally avoided saying "service objects" lately as nobody seems  to agree on what they mean. I used to use "service object" to describe any object that encapsulates high level business logic and is more or less decoupled from the Rails stack.

As described in this pull request, these do sound like Method Objects, which are well-described in a number of places and have been around for a long time. On the other hand, a search for "service objects" find no results on sites like c2wiki or refactoring.com, and the various blog posts describing them are generally describing different things.

Thoughts on using "method object" instead?